pkg/controller/issueapi/issue.go

```diff
@@ -84,27 +99,28 @@ func (ic *IssueAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if request.SymptomDate != "" {
 		if parsed, err := time.Parse("2006-01-02", request.SymptomDate); err != nil {
 			ic.logger.Errorf("time.Parse: %v", err)
-			controller.WriteJSON(w, http.StatusOK, api.Error("invalid test date: %v", err))
+			controller.WriteJSON(w, http.StatusUnprocessableEntity, api.Error("invalid test date: %v", err))
```

Comment:
change error text to symptom onset date - I missed that when we changed the date field

Reply:
Done